### DISCUSSION TITLE
Fill Online players in top islands menu

### DISCUSSION THREAD

**Discussion by Zsomi** [BG-Software-LLC/SuperiorSkyblock2#1319] [Implemented Feature Requests] [answered]
I imagined such a small trick that when you open the best islands menu (/is top), it should not only display the islands of 10 players who are on the top list but all available players who have an island. Obviously, whoever is in the first 10 places would be on the first menu page.

A few pictures of the idea:

![kép_2022-08-15_064843284](https://user-images.githubusercontent.com/31018282/184578476-3208f119-b4e7-4b33-bdc3-d157ac5ed878.png)

**Comment by OmerBenGera** [comment, marked as answer]
You can edit the top islands menu to include more islands
